packages/paste-core/components/chat-log/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@twilio-paste/chat-log",
+  "version": "0.0.0",
+  "category": "data display",
+  "status": "alpha",
+  "description": "Chat log is a collection of components that allow users to read a series of messages over time.",
+  "author": "Twilio Inc.",
+  "license": "MIT",
+  "main:dev": "src/index.tsx",
+  "main": "dist/index.js",
+  "module": "dist/index.es.js",
+  "types": "dist/index.d.ts",
+  "sideEffects": false,
+  "publishConfig": {
+    "access": "public"
+  },
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "yarn clean && NODE_ENV=production node build.js && tsc",
+    "build:js": "NODE_ENV=development node build.js",
+    "build:props": "typedoc --tsconfig ./tsconfig.json --json ./dist/prop-types.json",
+    "clean": "rm -rf ./dist",
+    "tsc": "tsc"
+  },
+  "peerDependencies": {
+    "@twilio-paste/box": "^5.0.0",
```

Comment:
issue: we really need a way to automate this, but be sure to follow the guidance on peer dependencies in the repo root, to ensure we have all the correct dependencies listed. https://github.com/twilio-labs/paste#peer-deps